linkedevents/events.php

```diff
@@ -1,13 +1,16 @@
 <?php
   foreach ($data->events as $event) {
     $result = "";
+    $siteUrlBase = "http://localhost/";
```

Comment:
There is a way to get this information from wordpress database using a function so it doesn't need to be hardcoded.
Maybe one of these?
[site url](https://developer.wordpress.org/reference/functions/site_url/)
[home url](https://developer.wordpress.org/reference/functions/get_home_url/)